### DISCUSSION TITLE
how to download the using wget, it failing

### DISCUSSION THREAD

**Discussion by thrilochanp** [jgm/pandoc#10030] [Q&A] [answered]
[ec2-user@ip-11-10-32-96 ~]$ wget https://github.com/jgm/pandoc/releases/download/3.2.1/pandoc-3.2.1-1.el7.x86_64.rpm
--2024-07-25 08:26:15--  https://github.com/jgm/pandoc/releases/download/3.2.1/pandoc-3.2.1-1.el7.x86_64.rpm
Resolving github.com (github.com)... 140.82.114.4
Connecting to github.com (github.com)|140.82.114.4|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-07-25 08:26:15 ERROR 404: Not Found.

**Comment by tarleb** [comment]
See here for a list of available files:
https://github.com/jgm/pandoc/releases/tag/3.2.1
There are no official rpm packages.

I wonder where you got this URL from?

**Reply by thrilochanp** [reply]
i would like to install on RHEL linux machine, can you pls provide the steps if you have handy

**Comment by tarleb** [comment, marked as answer]
There's a package conversion tool called "alien". You could try to
download a deb file and to then install that with alien.

**Reply by thrilochanp** [reply]
Thank you
